packages/libnvme/libnvme.spec

```diff
@@ -21,7 +21,7 @@ Requires: %{_cross_os}libnvme
 %{summary}.
 
 %prep
-%autosetup -n libnvme-%{version} -p1
+%autosetup -n libnvme-%{version}
```

Comment:
The commit message vs the change doesn't seem to make sense.

Reply:
Needed a rebase. Thanks for pointing it out.